config/routes.rb

```diff
@@ -29,12 +29,12 @@
       resources :participants, only: [:index]
     end
   end
+  put "newsletter_signup", to: "newsletter_signups#create"
+  match "*path/newsletter_signup", to: "newsletter_signups#create", via: :put
```

Comment:
These changes seem unrelated to volunteers... suggest only changing one thing at time, even if this does just feel like cleanup... Also, we just be using a resources block for this.

Reply:
@ColinW520 Hmm. I ended up with a merge conflict after merging those PRs last Friday. Not sure why that is coming in as an addition since it should already be in master. I'll look into what happened...